src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -404,7 +410,13 @@ function InitializeNetworkFiltering(frb: FunctionsRuntimeBundle): void {
         if (bundle.name === "google-gax") {
           const cs = newed.constructSettings;
           newed.constructSettings = (...csArgs: any[]) => {
-            (csArgs[3] as any).authorization = "Bearer owner";
+            // TODO: It doesn't seem like we need this anymore ... but maybe we do
```

Comment:
cc @abeisgoat what do you think about this?  Everything works for me without this.